src/work-allocation-2/components/find-person/find-person.component.ts

```diff
@@ -14,10 +16,10 @@ import { FindAPersonService } from '../../services/find-person.service';
 export class FindPersonComponent implements OnInit {
   @Output() public personSelected = new EventEmitter<Person>();
   @Input() public title: string;
-  @Input() public domainString: string = 'BOTH';
+  @Input() public domainString: string = 'All';
```

Comment:
As mentioned on Slack, giving the input as a string was causing issues in common-lib, could we not pass in the enum?

Reply:
as show on demo

Reply:
Can't we use PersonDomain.ALL?